website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,437 @@
+
+Delete any usages of <code>[application:didRegisterUserNotificationSettings:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623022-application?language=objc)</code> and <code>RCTPushNotificationManager</code>’s corresponding <code>didRegisterUserNotificationSettings:</code> as well.
+
+**Example:** See the RNTester <code>[AppDelegate.mm](https://github.com/facebook/react-native/blob/main/packages/rn-tester/RNTester/AppDelegate.mm)</code>.
+
+#### JS
+
+1. Remove any references to `alertAction`.
+2. Remove the `handler` argument on any calls to `removeEventListener`.
+3. Replace any usages of `repeatInterval` by firing multiple notifications using `fireDate` or `fireIntervalSeconds` instead.
+4. Note that `soundName` will be null when it is accessed on a `Notification` returned from `getScheduledLocalNotifications()` and `getDeliveredNotifications()`.
+
+</details>
+
+### Other Breaking Changes
+
+These are some of the key breaking changes in 0.74. Please consult the [full changelog](https://github.com/facebook/react-native/blob/main/CHANGELOG.md) for the complete list of breaking changes.
+
+- Remove web performance logging from `GlobalPerformanceLogger` ([#41299](https://github.com/facebook/react-native/pull/41299)).
+- Make `start/end` in styles always refer to writing direction ([#42251](https://github.com/facebook/react-native/pull/42251)).
+- **Android**: Removal of `JSIModule*` from `FabricUIManagerProvider` ([#42059](https://github.com/facebook/react-native/pull/42059)).
+  - This API was unused in open source — use [TurboModules](https://github.com/reactwg/react-native-new-architecture/blob/main/docs/turbo-modules.md) instead.
+- **Android**: Deprecation of `UIManagerModule.showPopupMenu` and `UIManagerModule.dismissPopupMenu` ([#42441](https://github.com/facebook/react-native/pull/42441))
+  - This API has been moved to the `@react-native/popup-menu-android` NPM package and will be removed in 0.75
+- **iOS**: Delete `configFilename` and `configKey` arguments from iOS codegen CLI ([#41533](https://github.com/facebook/react-native/pull/41533)).
+- **iOS**: Changed how the [`bundleURL`](https://github.com/facebook/react-native/pull/43994) is handled.
+  - Before, this was set when React Native was started in an instance variable and it was not possible to update it. Now, it takes a function (usually [`bundleURL`](https://github.com/facebook/react-native/blob/0.74-stable/packages/react-native/template/ios/HelloWorld/AppDelegate.mm#L22-L29) in AppDelegate) so the URL is re-evaluated when needed. This unblocks the possibility to update `bundleURL` across refreshes. This should affect your app only if you were changing the `bundleURL` variable after the app is started. In this case, move the logic that updates the variable to the `bundleURL` function in the `AppDelegate`.
```

Comment:
```suggestion
  - Before, this was set when React Native was started in an instance variable and it was not possible to update it.
  - Now, it takes a function (usually [`bundleURL`](https://github.com/facebook/react-native/blob/0.74-stable/packages/react-native/template/ios/HelloWorld/AppDelegate.mm#L22-L29) in AppDelegate) so the URL is re-evaluated when needed. 
  - It unblocks the possibility to update `bundleURL` across refreshes. 
  - It should affect your app only if you were changing the `bundleURL` variable after the app is started. In this case, move the logic that updates the variable to the `bundleURL` function in the `AppDelegate`.
```

I think it will be nicer to read it if we structure it with more points